src/policy/compiler.rs

```diff
@@ -470,8 +521,8 @@ impl<Pk: MiniscriptKey, Ctx: ScriptContext> AstElemExt<Pk, Ctx> {
         r: &AstElemExt<Pk, Ctx>,
     ) -> Result<AstElemExt<Pk, Ctx>, types::Error<Pk, Ctx>> {
         let lookup_ext = |n| match n {
-            0 => Some(l.comp_ext_data),
-            1 => Some(r.comp_ext_data),
+            0 => Some(l.comp_ext_data.clone()),
```

Comment:
Why do you need .clone() here? `CompExtData` is already `Copy`

Reply:
In the previous version having `CompilerExtData<Ctx: ScriptKey>`, the closure signature changes to `FnOnce` since it moves the variable `l` out of its environment. [This SO post](https://stackoverflow.com/questions/31371027/copy-trait-and-phantomdata-should-this-really-move) answers for this issue.

![2022 04 11-05 49 33 screenshot](https://user-images.githubusercontent.com/53688395/162646438-a3d9b2a6-2f89-4fa7-b6b9-ced2de96c90c.png)